src/Renderer/ThreeExtended/OBB.js

```diff
@@ -103,119 +105,28 @@ for (let i = 0; i < 9; i++) {
 }
 
 // get oriented bounding box of tile
-OBB.extentToOBB = function extentToOBB(extent, minHeight = 0, maxHeight = 0) {
+const builder = new BuilderEllipsoidTile();
+OBB.extent_R4326_ToOBB_4978 = function _extent_R4326_ToOBB_4978(extent, minHeight = 0, maxHeight = 0) {
```

Comment:
You don't need to rename the function; but you need to add a check at the beginning to validate `extent._internalStorageUnit`

Reply:
ok

Reply:
done